test/models/developer_test.rb

```
@@ -18,4 +18,20 @@ class DeveloperTest < ActiveSupport::TestCase
     assert_equal exp_commit_hashes, actuals
   end
 
+  test 'sanitize email' do
+    testEmail = Developer.sanitize_validate_email "dglazkov+test@gmail.com"
+    assert_equal testEmail, "dglazkov@gmail.com"
+  end
+
+  test 'sanitize wrong email' do
+    testEmail = Developer.sanitize_validate_email "dglazkov+test@google.com"
+    assert_equal testEmail, "dglazkov@chromium.org"
+  end
+
+  test 'sanitize non-email' do
```

This test should not return "commits.initial". If something is not an email address you should throw an exception.